cluster/juju/layers/kubernetes-worker/sos-scripts/kubectl

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -ux
+
+sudo -iu ubuntu kubectl cluster-info > $SOS_SCRIPT_DIR/cluster-info
+sudo -iu ubuntu kubectl cluster-info dump > $SOS_SCRIPT_DIR/cluster-info-dump
+for obj in pods svc ingress secrets pv pvc rc; do
+  sudo -iu ubuntu kubectl describe $obj --all-namespaces > $SOS_SCRIPT_DIR/describe-$obj
```

Comment:
It's a different file for each object type, `describe-$obj`, so they shouldn't overwrite eachother. Here was the resulting tree after running it:

```
├── kubectl
│   ├── cluster-info
│   ├── cluster-info-dump
│   ├── describe-ingress
│   ├── describe-nodes
│   ├── describe-pods
│   ├── describe-pv
│   ├── describe-pvc
│   ├── describe-rc
│   ├── describe-secrets
│   ├── describe-svc
│   ├── stderr
│   └── stdout
```

I think it's fine - or did I misinterpret what you're saying?